### DISCUSSION TITLE
Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230816

### DISCUSSION THREAD

**Discussion by brkirch** [brkirch/stable-diffusion-webui#30] [Announcements]
This is an offline standalone installer for a modified version of AUTOMATIC1111's Stable Diffusion web UI that includes experimental changes and uses the nightly PyTorch build from 8/14. Note that this is an test build so crashes and bugs should be expected. All dependencies needed to start using Stable Diffusion (other than the model) are included and this does not require Xcode command line tools to be installed.

This release currently only has the installer for Apple silicon Macs, an installer for Intel Macs will be included with 1.6.0 RC. macOS 13.4 or later is highly recommended, older versions of macOS are untested.

AUTOMATIC1111/stable-diffusion-webui#12599 is included, which should slightly reduce memory usage.

<hr /><em>This discussion was created from the release <a href='https://github.com/brkirch/stable-diffusion-webui/releases/tag/20230816_experimental'>Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230816</a>.</em>

**Comment by fckbauer** [comment]
do we need to update our older versions manually? or it updates automatically via extensions? Thanks

**Reply by brkirch** [reply]
To update you need to download and open the new installer, click `Update Existing Install`, and then choose the previously installed folder.

**Reply by fckbauer** [reply]
Thanks! 👌🏻

**Comment by Lassinger** [comment]
hi,

i would like to use img2img with batch....
what must be the directories for "input" and "output" here...?

many THX, Lassinger

**Comment by misato2099** [comment]
When I tried to run preprocessor of depth_zoe in Controlnet, it showed the following error:

  File "./stable-diffusion-webui/venv/lib/python3.10/site-packages/torch/nn/functional.py", line 3926, in interpolate
    raise TypeError(
TypeError: expected size to be one of int or Tuple[int] or Tuple[int, int] or Tuple[int, int, int], but got size with types [<class 'numpy.int64'>, <class 'numpy.int64'>]

There is no such error in the older version, and then preprocessor depth_midas works normally. Thank you.
